courseraresearchexports/commands/containers.py

```diff
@@ -14,66 +14,81 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from courseraresearchexports.containers import client
-from courseraresearchexports import utils
+from courseraresearchexports.containers import client, utils
+from courseraresearchexports.utils import docker_client
 from datetime import datetime
+from tabulate import tabulate
+import logging
+import sys
 
 
 def create_container(args):
```

Comment:
would it be possible to make this method take in its argument directly? and use camel_case